<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAccessCard</class>
 <widget class="QMainWindow" name="FrmAccessCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>715</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>324</width>
    <height>715</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Access Card</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ACCESS CARD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Txt_id_AccessCard">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>60</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="idUserLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Id Access card</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Fasilitas 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="Cmb_Fasilitas1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fasilitas1Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Id Fasilitas</string>
     </property>
    </widget>
    <widget class="QLabel" name="idUserLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Nama</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_Nama1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="idUserLabel_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Opening Hours</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_Operation1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>Cmb_Fasilitas1</zorder>
    <zorder>fasilitas1Label</zorder>
    <zorder>idUserLabel_2</zorder>
    <zorder>Lbl_Nama1</zorder>
    <zorder>idUserLabel_6</zorder>
    <zorder>Lbl_Operation1</zorder>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Fasilitas 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="idUserLabel_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Nama</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_Nama2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="idUserLabel_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Opening Hours</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_Operation2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="fasilitas2Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Id Fasilitas </string>
     </property>
    </widget>
    <widget class="QComboBox" name="Cmb_Fasilitas2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>idUserLabel_9</zorder>
    <zorder>Lbl_Nama2</zorder>
    <zorder>idUserLabel_10</zorder>
    <zorder>Lbl_Operation2</zorder>
    <zorder>fasilitas2Label</zorder>
    <zorder>Cmb_Fasilitas2</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Fasilitas 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="idUserLabel_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>83</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Nama</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_Nama3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="idUserLabel_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Opening Hours</string>
     </property>
    </widget>
    <widget class="QLabel" name="Lbl_Operation3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="fasilitas2Label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Id Fasilitas </string>
     </property>
    </widget>
    <widget class="QComboBox" name="Cmb_Fasilitas3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>idUserLabel_11</zorder>
    <zorder>Lbl_Nama3</zorder>
    <zorder>idUserLabel_12</zorder>
    <zorder>Lbl_Operation3</zorder>
    <zorder>fasilitas2Label_2</zorder>
    <zorder>Cmb_Fasilitas3</zorder>
   </widget>
   <widget class="QPushButton" name="PB_Submit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>301</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="PB_Fasilitas">
       <property name="text">
        <string>Fasilitas Detail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_Refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="PB_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_del">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
